tests/test_interfaces.py

```diff
@@ -0,0 +1,779 @@
+# pylint: skip-file
+""" This module tests the interface module """
+
+import unittest
+import mock
+import wifiphisher.common.interfaces as interfaces
+import pyric
+
```

Comment:
instead of doing that for all of our tests we should use nose or some similar tool. All you have to do is run nose from the `wifiphisher` directory.